bot.py

```diff
@@ -70,10 +75,17 @@
 pbCooldown = 100
 polling_list = dict()
 
+#Initialization
+print 'Starting boot sequence'
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
```

Comment:
Create/access an environment variable. Have a look at braxtons refactor for a way of how to do this, its how hes currently getting the PB_TOKEN. 

As for creds please message me directly and we can discuss it.